app/assets/stylesheets/wiki.css

```diff
@@ -68,9 +68,9 @@ a.navbar-offset {
     top: -100px;
     visibility: hidden;
 }
+
 @media (max-width: 768px) {
-  .btn.btn-primary {
```

Comment:
I'm still wondering if we need to look at precisely what element had been targeted when this was originally added. We may need to ensure the original fix is preserved too, you know? You can even ask the person who opened that PR.

Reply:
Previous fix was for `Add` button which is being restored as well